dotCMS/src/main/java/com/dotcms/browser/BrowserAPIImpl.java

```diff
@@ -265,15 +270,15 @@ private List<Map<String, Object>> filterReturnList(final BrowserQuery browserQue
     }
 
     /**
-     * Generates both the SQL query and the Lucene query that will be used to search for contents under a given folder
+     * Generates the SQL query that will be used to search for contents under a given folder
      * path and filtered by the criteria specified via the {@link BrowserQuery} parameter.
      *
      * @param browserQuery The filtering criteria set via the {@link BrowserQuery}.
      *
-     * @return The {@link Tuple3} object containing (1) the SQL query, (2) the Lucene query, and (3) the parameters that
+     * @return The {@link Tuple3} object containing (1) the SQL query, and (2) the parameters that
      * must be provided for the queries.
      */
-    private Tuple3<String,String, List<Object>> selectQuery(final BrowserQuery browserQuery) {
```

Comment:
Fix the cognitive complexity issue reported by Sonarqube for this method.   Can use Copilot chat or chatgpt to help break this down or manually break down into extra private methods

Reply:
Done